dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HadoopUtils.java

```diff
@@ -435,9 +432,7 @@ public ExecutionStatus getApplicationStatus(String applicationId) throws Excepti
         } else {
             //may be in job history
             String jobHistoryUrl = getJobHistoryUrl(applicationId);
-            if (logger.isDebugEnabled()) {
-                logger.debug("generate yarn job history application url, jobHistoryUrl={}", jobHistoryUrl);
-            }
+            logger.info("jobHistoryUrl={}", jobHistoryUrl);
```

Comment:
The outer loop calls this method, and if the log level is changed to INFO here, This information will always be printed(the YARN task runtime)

Reply:
Sorry, I did not modify the code, it may be caused by a git conflict

Reply:
It is suggested to pull the latest branch code, modify and submit PR again

Reply:
ok,Thanks for your code review very much.